modules/ROOT/pages/configuration/user/user_provisioning_api.adoc

```diff
@@ -147,7 +147,7 @@ sending a Basic HTTP Authorization header.
 [cols=",,",options="header",]
 |==============================================================
 |Request Path |Method |Content Type
-|`Syntax: ocs/v1.php/cloud/users/{userid}` |`GET` |`text/plain`
+| `Syntax: ocs/v1.php/cloud/users/\{userid\}` | `GET` | `text/plain`
```

Comment:
You are using a blank between `|`` and the next character, this is good and improves readability !
Also used in other tables like occ commands.
To be consistent, you should do that on all following tables too...

Reply:
That somehow slipped in. It was meant to only be a part of #143.

Reply:
without double checking, is this then addressed in #143 ?
If not pls add there as this PR is mergen now

Reply:
Yep, the change is there. As I said, this crept through, by accident.